wallet2/db/db.go

```diff
@@ -0,0 +1,549 @@
+package db
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"strings"
+	"time"
+
+	//nolint:all
+	_ "github.com/glebarez/go-sqlite"
+)
+
+const (
+	AddressTable     = "addresses"
+	TransactionTable = "transactions"
+	PairTable        = "pairs"
+)
+
+type DB interface {
```

Comment:
Do we need to define interface here?

Reply:
it would be better to have, because we have some private methods which we don't want to expose them to higher layer of application plus for tests we don't have any problem because `newDB` method has input which we can pass `memory` then run all tests.